media-libs/noise-suppression-for-voice/noise-suppression-for-voice-1.10-r2.ebuild

```diff
@@ -40,6 +40,9 @@ RDEPEND="${DEPEND}"
 
 PATCHES=(
 	"${FILESDIR}/${P}-tests.patch"
+	"${FILESDIR}/${P}-Patch-out-use-of-execinfo.h.patch"
+	"${FILESDIR}/${P}-Patch-out-use-of-stat64.patch"
+	"${FILESDIR}/${P}-Patch-out-uses-of-nl_langinfo-for-musl.patch"
```

Comment:
Have any of these been submitted upstream?

Reply:
I haven't actually checked earlier, but apparently yes: https://github.com/juce-framework/JUCE/pull/1239.
Not sure whether I should duplicate the changes from that PR instead of these patches (they are slightly different, the ones in the PR are a bit less heavy-handed with patching out `execinfo.h` and `stat64` for example). We should probably at least link to that PR.

Reply:
OK, can we link to the PR then?

Reply:
Apologies for the delay, I've added a comment linking to the JUCE PR.